frontend/tests/e2e-tests.sh

```
@@ -71,27 +76,35 @@ if [[ " ${SCRIPT_SHORT_ARGS[@]} " =~ " -h " ]] || [[ " ${SCRIPT_LONG_ARGS[@]} "
+try :
+	s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+	# Using python to check if the port is opened is safer for compatibility
+	# But it increase the execution time of the script with the timeout
+	# A local TCP connection should never reach a 1s delay
+	s.settimeout(1)
+	s.connect(('127.0.0.1', $BACKEND_PORT))
+	port_already_in_used = True
+except :
+	port_already_in_used = False
+s.close()
+# We use "not" because 0 is True and 1 is False in bash
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)